template.hbr

```diff
@@ -204,11 +210,21 @@ players:
                     "font_style": null,
                     "font_weight": 700
                 },
+                "enum": {
+                    "color": "#{{maroon}}",
```

Comment:
```suggestion
                    "color": "#{{teal}}",
```

As per VSCode + JetBrains, will update the style guide after this.

![image](https://github.com/catppuccin/zed/assets/58985301/8c6667bf-8fc8-4367-b50b-98c4b5c6df85)